src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_test.py

```diff
@@ -1190,8 +1190,11 @@ def test_oom_crash(self, mock_stdout):
     self.fs.CreateDirectory('/fake/corpus_oom')
     os.environ['FUZZ_CORPUS_DIR'] = '/fake/corpus_oom'
 
-    with mock.patch('subprocess.Popen',
-                    create_mock_popen(self.oom_output)) as mock_popen:
+    new_units_added = 11
+    with mock.patch(
+        'subprocess.Popen',
+        create_mock_popen(self.oom_output, '/fake/inputs-disk/temp-1337/new',
+                          None, new_units_added)) as mock_popen:
```

Comment:
None is used to show

              'new_units_added':
                  0,	                  0,

which just means we tried merge (since new_testcase_directory has units), but then running with -merge=1 failed, so units=0.